package.json

```diff
@@ -83,6 +86,9 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
+    "@gandi/babel-preset-gettext": "*",
+    "babel-cli": "*",
```

Comment:
:+1: We discussed in #5 and I provided a fix in #6 so it should be fixed on rebase.